presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -353,38 +353,38 @@ else if (leftNDV <= matchingRightNDV) {
 
     @VisibleForTesting
     PlanNodeStatsEstimate addJoinComplementStats(
-            PlanNodeStatsEstimate sourceStats,
-            PlanNodeStatsEstimate baseJoinStats,
+            PlanNodeStatsEstimate leftStats,
+            PlanNodeStatsEstimate innerJoinStats,
```

Comment:
Continuing the discussion from https://github.com/prestodb/presto/pull/11856#discussion_r232167964

I like the old names better. Do you feel strongly about this rename?